warehouse/core/src/main/java/datawave/iterators/filter/ConfigurableAgeOffFilter.java

```diff
@@ -327,56 +327,47 @@ public void init(SortedKeyValueIterator<Key,Value> source, Map<String,String> op
     }
     
     /**
-     * This method instantiates the the necessary implementations of the {@code Filter} interface, as they are defined in the configuration file specified by
+     * This method instantiates the necessary implementations of the {@code Filter} interface, as they are defined in the configuration file specified by
      * {@code this.filename}.
      *
      * @throws IllegalArgumentException
      *             if there is an error in the configuration file
      * @throws IOException
      */
-    private void initFilterRules() throws IllegalArgumentException, IOException {
+    private synchronized void initFilterRules() throws IllegalArgumentException, IOException {
```

Comment:
This changes the existing synchronization from class-level scope to instance-level. Class scope is still needed here. Maybe the method can be made static?

Reply:
Changing it to `private static synchronized ...` requires the `getLongProperty()` to also be static, which cascades a lot more changes.
I moved the `synchronized` block back down into the code and reworked it to remove it from the blocker level problems.